plugins/database/src/low_level_odbc.cpp

```diff
@@ -572,7 +578,7 @@ cllExecSqlWithResult( icatSessionStruct *icss, int *stmtNum, const char *sql ) {
        needed here, and in fact causes postgres processes to be in the
        'idle in transaction' state which prevents some operations (such as
        backup).  So this was removed. */
-    rodsLog( LOG_DEBUG10, "%s", sql );
+    log_sql::debug("{}", sql);
```

Comment:
Consider including something like `"{}: SQL = [{}]", __func__, sql` to help admins understand what is being logged.